src/main/java/mate/academy/spring/service/impl/UserServiceImpl.java

```diff
@@ -0,0 +1,24 @@
+package mate.academy.spring.service.impl;
+
+import java.util.List;
+import mate.academy.spring.dao.UserDao;
+import mate.academy.spring.model.User;
+import mate.academy.spring.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class UserServiceImpl implements UserService {
+    @Autowired
+    private UserDao userDao;
```

Comment:
The best practice is to use injection through the constructor neither through the field.

Reply:
I agree